lib/common/asar.js

```diff
@@ -582,8 +581,7 @@
       };
       mkdirSync = fs.mkdirSync;
       fs.mkdirSync = function(p, mode) {
-        var filePath, isAsar, ref;
-        ref = splitPath(p), isAsar = ref[0], filePath = ref[2];
+        const [isAsar, , filePath] = splitPath(p);
```

Comment:
I didn't know `const [isAsar, , filePath] = splitPath(p);` would work. That's cool.

I just tried it in the node (5.8) repl, but destructuring assignment is [not ready yet](https://github.com/nodejs/node/issues/3101). Is there an electron repl? That would be cool. I tried it in [Mancy](http://mancy-re.pl/) too, but no :game_die: because even though it's an electron app I think it's just using node under the hood.

![screen shot 2016-03-22 at 12 19 37 pm](https://cloud.githubusercontent.com/assets/2289/13964741/72ff1ff0-f028-11e5-9711-3d0b9dcb364a.png)